lib/chef/resource/windows_feature_dism.rb

```diff
@@ -55,7 +55,7 @@ class WindowsFeatureDism < Chef::Resource
         unless features_to_install.empty?
           message = "install Windows feature#{'s' if features_to_install.count > 1} #{features_to_install.join(',')}"
           converge_by(message) do
-            install_command = "#{dism} /online /enable-feature #{features_to_install.map { |f| "/featurename:#{f}" }.join(' ')} /norestart"
+            install_command = "dism.exe /online /enable-feature #{features_to_install.map { |f| "/featurename:#{f}" }.join(' ')} /norestart"
```

Comment:
is that deliberate?  reads like a step backwards

Reply:
the dism method was from the cookbook and came over when I was diffing them by accident. It was to support funky 32bit vs. 64bit stuff. We discussed a while back that we could ditch this logic.